provisioning/playbooks/roles/grpc/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+
+- name: install grpc base dependencies
+  apt: name={{item}}
+  become: yes
+  become_user: root
+  with_items:
+      - python-dev
+      - python-pip
+      - libtool
+      - autoconf
+
+- name: update pip
```

Comment:
pip gets installed in that apt step above.  But I agree about splitting it out into a role.  Maybe along with python-dev and python-virtualenv?